private/src/styles/blocks/button/styles/_default.scss

```
@@ -38,3 +38,7 @@
     color: var(--wp--preset--color--black)
   }
 }
+
+.wp-block-button .wp-block-button__link:hover .icon-arrow-left {
+  @include icon(213px, 240px, 16px, 16px);
```

This re-adds all of the styles that don't need redeclaring. To change the background position only, use:

```suggestion
  @include icon-position(213px, 240px);
```